nova/tests/unit/db/test_migrations.py

```diff
@@ -1009,6 +1014,11 @@ def _check_390(self, engine, data):
         self.assertColumnExists(engine, 'shadow_instance_extra',
                                 'trusted_certs')
 
+    def _check_392(self, engine, data):
+        self.assertColumnExists(engine,
+                                'console_auth_tokens',
+                                'internal_access_path')
```

Comment:
This is not a real test, is it? It's true before and after the migration. Our migration is not "additive", so we would have to check for the change.

Reply:
Thanks! Updated.